content/v1.3/introduction/getting-started.md

```diff
@@ -1133,9 +1137,22 @@ We can use the `routes` helper method that is available in our views and actions
 We can make a similar change in `apps/web/controllers/books/create.rb`:
 
 ```ruby
+...
 redirect_to routes.books_path
+...
 ```
 
+and in `apps/web/templates/books/index.html.erb`:
+
+```erb
+...
+<a href="<%= routes.new_book_path %>">New book</a>
+...
+```
+
+(You could also extend this to the specs if you'd like,
```

Comment:
Do you think we need to parenthesis this statement?

Reply:
Removed parenthesis, and added _why_ that might be prefererd.

"You could also extend this to the specs if you'd like, but changing the path can affect users so you may want to repeat yourself there (to make a potentially breaking change for users harder to implement)."

I don't _love_ that this is the note we end on before the "Wrapping Up" section, since it's not directly related to Hanami.